host-controller/src/main/java/org/jboss/as/host/controller/model/host/HostResourceDefinition.java

```diff
@@ -160,6 +162,9 @@
             .setValidator(new StringLengthValidator(1, true))
             .setStorageRuntime()
             .build();
+    public static final SimpleAttributeDefinition ORGANIZATION_IDENTIFIER = SimpleAttributeDefinitionBuilder.create(ModelDescriptionConstants.ORGANIZATION, ModelType.STRING, true)
+            .setValidator(new StringLengthValidator(1, true))
+            .build();
```

Comment:
Should this be on DomainRootDefinition?

In theory different hosts could have different "organizations" since it's a term that means what the user wants. It would be annoying to have to set it in every host.xml though. I'm not sure how hard it would be to have both here and DomainRootDefinition, with the DomainRootDefinition value only being used in a host/domain server report if the host level value wasn't set.